packages/modules/imodel-browser/src/containers/ITwinGrid/useITwinData.ts

```diff
@@ -7,31 +7,34 @@ import React from "react";
 import {
   ApiOverrides,
   DataStatus,
-  ProjectFilterOptions,
-  ProjectFull,
+  ITwinFilterOptions,
+  ITwinFull,
+  ITwinType,
 } from "../../types";
 import { _getAPIServer } from "../../utils/_apiOverrides";
-import { useProjectFilter } from "./useProjectFilter";
+import { useITwinFilter } from "./useITwinFilter";
 
 export interface ProjectDataHookOptions {
   requestType?: "favorites" | "recents" | "";
+  itwinType?: ITwinType;
   accessToken?: string | undefined;
-  apiOverrides?: ApiOverrides<ProjectFull[]>;
-  filterOptions?: ProjectFilterOptions;
+  apiOverrides?: ApiOverrides<ITwinFull[]>;
+  filterOptions?: ITwinFilterOptions;
 }
 
 const PAGE_SIZE = 100;
 
-export const useProjectData = ({
+export const useITwinData = ({
   requestType = "",
+  itwinType = "Project",
```

Comment:
This should be called itwinSubClass, right? Otherwise it could be confused with the iTwin's `type` property, which it isn't....

Reply:
Done